roles/xmpp/tasks/prosody.yml

```diff
@@ -26,7 +26,7 @@
   file: state=directory path=/decrypted/prosody owner=prosody group=prosody
 
 - name: Configure Prosody
-  template: src=prosody.cfg.lua.j2 dest=/etc/prosody/prosody.cfg.lua group=root owner=root
+  template: src=prosody.cfg.lua.j2 dest=/etc/prosody/prosody.cfg.lua group=prosody owner=prosody
```

Comment:
A better solution is to make the file readable by prosody (i.e. world readable) but keep root ownership.
If prosody owns its config file it can write to it (which is something it should‘nt be allowed to do)

Reply:
I'll test it again, but I think prosody still had issues with letting root own the file. Maybe prosody needs write access? Very strange.

You're suggesting `owner=root group=prosody mode=0664`  or something different?

Reply:
I was wrong, making it world readable works, although I don't totally understand why.